.storybook/assets/stylesheets/index.scss

```diff
@@ -4,6 +4,7 @@
 
 @use '../../../packages/web/src/components/Button';
 @use '../../../packages/web/src/components/Container';
+@use '../../../packages/web/src/components/CheckboxField';
```

Comment:
Please sort alphabetically (in English).

Reply:
Fixed.